### ISSUE TITLE
Publishing code with lots of LogManager debug calls

### ISSUE THREAD

**Issue by tomluchowski** [OpenDungeons/OpenDungeons#928]
As in title , suppose I have a code decorated with various calls to LogManager::logMessage() . 
What should be done with it when publishing on remote server : Should it be either commented out for future use or removed so the code looks nicer ?

**Comment by hwoarangmy** [comment]
There are 2 cases:
- If the log is really specific for your setup (for example for finding some magic number), you can remove it
- If the log may be interesting for debug reasons, you can let it in the code with OD_LOG_DBG. This way, the log will not be printed unless you explicitly turn it on. Note that it may be tempting to let all your logs but please keep in mind that too many logs make the logs unreadable.

In any case, you should not let commented logs IMHO. However, your code should be well written enough to allow someone who would debug it to do so without having to guess too much.

**Comment by tomluchowski** [comment]
Real Example : 

```
    std::stringstream ss;
    ss << "Current pitch: " << pitch << ", desired pitch: " << mCameraPitchDestination << std::endl;
    ss << "Pitch update: " << pitchChange * frameTime << std::endl;
    ss << "Current roll: " << roll << ", desired roll: " << mCameraRollDestination << std::endl;
    ss << "Roll update: " << rollChange * frameTime << std::endl;
    OD_LOG_INF(ss.str());
```

Should the lines starting with ss<< be left, commented out or removed ?

**Comment by hwoarangmy** [comment]
Well, it was already there and it may be usefull for debugging. I would say to let it commented. However, please comment all the block (like it was) instead of line per line (it is not as easy to remove).
But we should avoit this kind of thing as much as possible. Camera is always a bit hard to understand when it starts getting crazy. So, in this particular case, it might be useful.

**Comment by Bertram25** [comment]
This has been more or less fixed with: 65b90a162384e5592a0d2c4cf42f4cf9e16705f6
